spec/oncocore_stage.txt

```diff
@@ -23,15 +23,15 @@ Description:    "A panel that contains cancer staging information. This class is
 0..0		    BodyLocation
 0..0            Device
 
-Abstract Element:  CancerStageClassification
+Abstract Element:  CancerStageCategory
 Based on:       SimpleCodedNonLaboratoryObservation
 Description:    "Abstract parent class for members of cancer staging panels. Cancer panel members must include a timing element and staging system, and focus on a cancer disorder. Specific realizations will have value sets specific to certain staging systems."
                 SubjectOfRecord is type PatientSubjectOfRecord
                 Performer is type PractitionerPerformer
 1..1            DataValue
 0..0            DataAbsentReason
 0..1            ref(PrimaryCancerCondition)
-0..1            CancerStageTiming 
+0..1            TimeOfClassification 
```

Comment:
Do we need to provide additional context in the class name, e.g. "CancerStagingTimeOfClassification"?

Reply:
Looks ok, I think. And will suffice.